src/main.cpp

```diff
@@ -24,46 +14,36 @@ Dispatcher *dispatcher;
 unsigned long lastPublish = 0;
 
 void publishMessage() {
-    long start, json_build_time;
```

Comment:
We haven't been making use of this functionality and it's become less useful over time because we're doing less in the "handle" functions of each sensor. I've left the underlying functions alone, but removed it from the main file for now until we find a better way to run benchmarks

Reply:
We could probably put the publish methods with the data queue since they're closely related anyway.

Reply:
True, I think I will leave it to another issue though as we were discussing publishing when the buffer gets (close to) full as opposed to on a set interval.